requirements-production.txt

```diff
@@ -19,7 +19,7 @@ cssutils==2.11.1
 html2text==2020.1.16
 tablib==3.5.0
 django-redis>=5.2.0,<5.3
-django-select2>=6.3.1,<6.4
+django-select2>=7.4.1,<8.3
```

Comment:
Minimum should be the current version, whatever that is. E.g.

>=8.2.x,<8.3 (whatever x currently is)

Sorry for the confusion.  No point allowing a previous version when we know 8.2.x works.